queries.sql

```diff
@@ -1,3 +1,51 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon';
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-01-01';
+SELECT name FROM animals WHERE neutered = true AND escape_attempts < 3;
+SELECT date_of_birth FROM animals WHERE name = 'Agumon' OR name='Pikachu';
+SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
+SELECT * FROM animals WHERE neutered = true;
+SELECT * FROM animals WHERE name != 'Gaburon';
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+-- edit Table 
+START TRANSACTION;
+ALTER TABLE animals RENAME COLUMN species TO unspecified;
+ROLLBACK;
+
+BEGIN TRANSACTION;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+COMMIT TRANSACTION;
+
+
+BEGIN;
+delete from animals;
+ROLLBACK;
+
```

Comment:
as required you are asked to verify that changes were made after the alter and undone after the rollback. as seen below

```
BEGIN;
delete from animals;
SELECT COUNT(*) FROM ANIMALS;
ROLLBACK;
SELECT COUNT(*) FROM ANIMALS;
```